src/generation.rs

```diff
@@ -5767,7 +5790,7 @@ fn add_struct_derives<T: DataType>(data_type: &mut T, used_in_key: bool, is_enum
             // there's no way to do non-derive() proc macros in the codegen
             // cate so we must sadly use a newline like this. codegen manages indentation
             data_type.derive(&format!(
-                "Derivative)]\n#[derivative({}",
+                "derivative::Derivative)]\n#[derivative({}",
                 key_derives(false)
```

Comment:
this was for consistency since the other non-std attributes (schemas::JsonSchema, serde::Serialize, serde::Deserialize) we derive are explicitly mentioned instead of an import